demo/IntrospectionModal.tsx

```diff
@@ -156,12 +188,33 @@ export class IntrospectionModal extends React.Component<IntrospectionModalProps>
     );
   }
 
+  renderURLTab() {
+    const { urlText, headers } = this.state.current;
+    return (
+      <>
+        <textarea
+          value={urlText || ''}
+          placeholder="Paste URL here"
+          onChange={this.handleURLChange}
+        />
+        <div>
+          Paste headers (in JSON format) into the textarea below.
```

Comment:
Great for PoC but it bad UX in the long run, it would be constant source of frustration for users.

Reply:
I believe this is an acceptable solution. Please take a look at GraphQL Playground. Users paste headers in GraphQL Playground in JSON format. GraphQL Playground is production-ready tool.

![graphqlbin](https://user-images.githubusercontent.com/7892779/73135415-a6505580-4052-11ea-961d-1bcc607eabe4.png)

Also users paste variables in GraphiQL and GraphQL Playground in JSON format too. The target audience of these tools are developers. So I think this is ok.